bits/spf_discovery/spf_discovery.py

```diff
@@ -0,0 +1,14 @@
+from typing import Iterator
+
+from octopoes.models import OOI
+from octopoes.models.ooi.dns.records import DNSTXTRecord
+from octopoes.models.ooi.email_security import DNSSPFRecord
+
+
+def run(
+    input_ooi: DNSTXTRecord,
+    additional_oois,
+) -> Iterator[OOI]:
+
+    if input_ooi.value.startswith("v=spf1"):
+        yield DNSSPFRecord(dns_txt_record=input_ooi.reference, value=input_ooi.value, ttl=input_ooi.ttl)
```

Comment:
Shouldn't it be the job of the normalizer to normalize a SPF TXT record to a DNSSPFRecord object?

Reply:
It might be actually. But I want to extend this bit with this code from internet.nl

https://github.com/internetstandards/Internet.nl/blob/main/checks/tasks/spf_parser.py

to parse the SPF record (includes, all etc), and create findings if parsing does not work.

Reply:
> Shouldn't it be the job of the normalizer to normalize a SPF TXT record to a DNSSPFRecord object?

Im ok with having a BIT do this work. If there is a TXT record starting with v=spf1 we can then also conclude that we need to have a DNSSPFRecord. Or, if we cant parse it, that we create a Finding about this parse-issue like Noam concludes.